# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
# Commit: 7057437ddf19
# Full Hash: 7057437ddf194b62a158ce84345802e8b2bc6ca8
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 6
# Description:
#   Bug 1654112 - Constify VideoConduit members that are based on prefs. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112848
# ==============================================================================

diff -r 01569f2b1141 -r 7057437ddf19 media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
--- a/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Tue Apr 06 18:29:30 2021 +0200
+++ b/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Wed Apr 07 09:44:33 2021 +0200
@@ -25,6 +25,7 @@
 #include "MediaTrackGraph.h"
 #include "MediaTrackListener.h"
 #include "MediaStreamTrack.h"
+#include "TaskQueueWrapper.h"
 #include "transportflow.h"
 #include "transportlayerloopback.h"
 #include "transportlayerdtls.h"